MTM-60386-Missing-info-about-required-permissions-to-use-Smart-Rules-relying-on-SMS-provider (#2462)
SoftwareAG/c8y-docs: content/cockpit/smart-rules-collection.md

If an alarm is created, an SMS is sent.

{{< c8y-admon-req >}}
This smart rule is only available if:

> - your tenant has configured an SMS provider

Reviewer: ```suggestion
- Your tenant has configured an SMS provider.
```

>  This rule is available when your tenant has: 
> - configured SMS provider
> - READ permission for the permission type "Option management"

Reviewer 2: I think that SMS provider requirement applies to the tenant, but READ permission applies to the user, so maybe better:

```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```
?

> - your user has a READ permission for the permission type "Option management"

Reviewer: ```suggestion
- Your user has a READ permission for the permission type "Option management".
```

If an alarm is created, sends email or SMS.

{{< c8y-admon-req >}}

>  This rule is available when your tenant has: 

Author: ```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```

> - your tenant has configured an SMS provider

Reviewer: ```suggestion
- Your tenant has configured an SMS provider.
```

> - READ permission for the permission type "Option management"

Reviewer 2: See above.

Author: Ok, will do the same.

>  This rule is available when your tenant has: 
> - configured SMS provider
> - READ permission for the permission type "Option management"

Author: ```suggestion
This smart rule is only available if:
- your tenant has configured an SMS provider
- your user has a READ permission for the permission type "Option management"
```

> - your user has a READ permission for the permission type "Option management"

Reviewer: ```suggestion
- Your user has a READ permission for the permission type "Option management".
```
